packages/react-i18n/src/i18n.ts

```diff
@@ -299,6 +363,31 @@ export class I18n {
     return WEEK_START_DAYS.get(country) || DEFAULT_WEEK_START_DAY;
   }
 
+  // Intl.NumberFormat sometimes annotates the "curerncy symbol" with a country code.
+  // For example, in locale 'fr-FR', 'USD' is given the "symbol" of " $US".
+  // This method strips out the country-code annotation, if there is one.
+  // (So, for 'fr-FR' and 'USD', the return value would be " $").
+  //
+  // For other currencies, e.g. CHF and OMR, the "symbol" is the ISO currency code.
+  // In those cases, we return the full currency code without stripping the country.
+  getShortCurrencySymbol(currencyCode?: string) {
```

Comment:
Should `getShortCurrencySymbol` have test coverage as well?

Reply:
If it's not public API, it should be marked `private` too

Reply:
Both good points.  Adjusted as suggested.